hive-agent/src/main/java/org/apache/ranger/authorization/hive/authorizer/RangerHiveAuditHandler.java

```diff
@@ -21,7 +21,6 @@
 
 import java.util.*;
 
-import org.apache.commons.collections.CollectionUtils;
```

Comment:
why did this class get removed, (which forces many of the changes in this PR?

Reply:
After updating the Hive version and regenerated the sources, the method isNotEmpty() in CollectionUtils and MapUtils, doesn't exist anymore.

CollectionUtils was only used to access isNotEmpty(). After removing all calls, the import was unused and I had to remove it to workaround mvn errors from a checkstyle plugin.

Reply:
@xBis7 Please add a comment to one of the java files explaining this so I don't forget.

Reply:
Done.